warehouse-management-service/db/migrations/000003_create_shelf_block_table.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS shelf_block(
```

Comment:
Should this have a relationship to it's warehouse?

Reply:
It should, I missed that. Added `warehouse_id`